### PR TITLE
Kudos

### LINKED ISSUE #80: Kudos
Створити реалізацію ставлення та прибирання kudos зареєстрованим користувачем

### PR DESCRIPTION
**Pull request by daniilievtukhov**
closed #80

### CODE REVIEW

**Review by whoisalex1W — APPROVED** [review]

File: src/components/TalentPage/components/ListProofs/components/ProofBlock/ProofBlock.jsx [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+import { Kudos } from "./components/Kudos";
 
 export function ProofBlock({ id, link, text, created }) {
     return (
         <div className={s.out}>
             <div className={s.proofs}>
                 <div className={s.info}>
                     <h1>Link:</h1>
-                    <a className={s.link} href={link} target="_blank" rel="noreferrer">
+                    <a className={s.link} href={link} target="_blank">
```
Comment (Ruslanchik01) [inline src/components/TalentPage/components/ListProofs/components/ProofBlock/ProofBlock.jsx]: return attribute rel="noreferrer"
```suggestion
                    <a className={s.link} href={link} target="_blank" rel="noreferrer">
```

File: src/components/TalentPage/components/ListProofs/components/ProofBlock/components/Kudos.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            })
+            .catch((err) => console.log(err));
+    }, []);
+
+    function likeProof() {
+        if (token) {
+            if (!isLiked) {
+                TalentsService.putKudos(id, token)
+                    .then((response) => {
+                        setKudos(kudos + 1);
```
Comment (Ruslanchik01) [inline src/components/TalentPage/components/ListProofs/components/ProofBlock/components/Kudos.jsx]: You should use previous state
```suggestion
                        setKudos((prev)=>prev+1);
```

File: src/components/TalentPage/components/ListProofs/components/ProofBlock/components/Kudos.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+                        setIsLiked(!isLiked);
```
Comment (Ruslanchik01) [inline src/components/TalentPage/components/ListProofs/components/ProofBlock/components/Kudos.jsx]: You should use previous state
```suggestion
                        setIsLiked((prev)=>!prev);
```

File: src/components/TalentPage/components/ListProofs/components/ProofBlock/components/Kudos.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                    })
+                    .catch((err) => console.log(err));
+            } else {
+                TalentsService.deleteKudos(id, token)
+                    .then(() => {
+                        setKudos(kudos - 1);
+                        setIsLiked(!isLiked);
```
Comment (Ruslanchik01) [inline src/components/TalentPage/components/ListProofs/components/ProofBlock/components/Kudos.jsx]: same problem like on 29, 31, and 38 rows

**Pull request merged**